src/clj_kondo/impl/config.clj

```diff
@@ -183,7 +183,8 @@
              ;; if below :linter-name is set to true, type (linter name) of reported the finding
              ;; is appended to the end of the default pattern as " [{{type}}]"
              :linter-name false
-             :canonical-paths false}}) ;; set to true to see absolute file paths and jar files
+             :canonical-paths false} ;; set to true to see absolute file paths and jar files
+    :min-clj-kondo-version "2019.10.26"}) 
```

Comment:
Why have this as the minimal clj-kondo version? I'd just comment this out, having no minimum clj-kondo version

Reply:
Happy to comment this out, was just looking for a sensible default value. I'll make these changes when I hear back on the other review comment.

Reply:
Fixed in 14cd00636de4c7fe531644a4edf198ae66241c98